iot_packet_verifier/src/pending_burns.rs

```diff
@@ -216,3 +332,22 @@ pub struct Burn {
     pub payer: PublicKeyBinary,
     pub amount: i64,
 }
+
+#[derive(Clone, FromRow, Debug)]
+pub struct BurnAttempt {
+    pub payer: PublicKeyBinary,
+    amount: i64,
+    latest_transaction_signature: String,
```

Comment:
insert date would be useful here, even if only to assist manual auditing, verification when browsing the table